spec/models/redirection_spec.rb

```diff
@@ -83,4 +83,32 @@
       expect(redirection.tag_uri).to eq(id)
     end
   end
+
+  describe "#unlink" do
+    it "destroys the redirection" do
+      redirection = create(:redirection)
+
+      redirection.unlink
+
+      expect(Redirection.find_by(id: redirection.id)).to be_nil
```

Comment:
If you don't like that, your solution is fine. This is Not That Big Of A Deal™

P.S. `be_falsey` is a good matcher too.